plugins/generator-1.21.1/neoforge-1.21.1/templates/gui/gui_container.java.ftl

```diff
@@ -311,4 +315,4 @@ public class ${name}Menu extends AbstractContainerMenu implements Supplier<Map<I
 
 }
 </#compress>
-<#-- @formatter:on -->
\ No newline at end of file
+<#-- @formatter:on -->

```

Comment:
Please revert those line-ending changes as they are not related to the PR. Thanks!